### DISCUSSION TITLE
I haven't been able to play the leaderboard for a long time

### DISCUSSION THREAD

**Discussion by Henry-Lee24** [cyberbotics/wrestling#83] [Q&A] [answered]
Why does someone spend more than 3 hours on the leaderboard. This has caused me to be unable to play on the leaderboard for a long time.Could you please take a look at the problem?

**Comment by Henry-Lee24** [comment]
Oh!That program has been running for 5 hours.

**Reply by JustLuckyRobotics** [reply]
its when their code crashes/something similar, the action will run for 6 hours until GitHub auto cancels.

**Reply by Henry-Lee24** [reply]
Yes, but they take up six hours each time they run, which is a waste of resources

**Comment by omichel** [comment]
Yes, it seems there is a problem with the controller of some participant.
We are investigating this now and will keep you posted.
Meanwhile, I cancelled manually some runs to unblock the queue, so that you should be able to run your robot controller now.

**Reply by Henry-Lee24** [reply]
Thank you!

**Reply by JustLuckyRobotics** [reply]
For this last week if this issue is not easily found/solved, is it possible to limit action times to 60 minutes? Even if they win the last game within that time, participants will recommit if they are winning I'm sure. This way bricked code will only brick the board for a shorter period. Even someone from the bottom of the board could climb to the top within 3-4 commits (approx. 20 matches in an hour).

**Reply by MJavadZallaghi** [reply]
@omichel As a participant, I'm worried about the queue on commits in the last day. It would be great if you design an execution policy for the controllers in the last day or or monitor the actions to provide the last change opportunity for all, in a justly way.

**Comment by omichel** [comment]
Yes, limiting the jobs to 60 minutes seems to be a good idea. I will implement it as soon as possible.

**Reply by omichel** [reply, marked as answer]
Done in 6abf867ffb78bcdf02a9a197d89c13c7bb855d13.
